src/xdocs/checks.xml

```diff
@@ -360,7 +360,8 @@
             <td><a href="config_design.html#InnerTypeLast">InnerTypeLast</a></td>
             <td>
               Checks nested (internal) classes/interfaces are declared at the bottom of the
-              primary (top-level) class after all method and field declarations.
+              primary (top-level) class after all init and static init blocks,
```

Comment:
I wonder why enums/annotations are not handled by this check? Perhaps this check should be reviewed one more time once records will be done.

Reply:
agree, but this cane be done later.